esp/templates/users/usersearch/list_selector.html

```diff
@@ -73,6 +73,14 @@
             <div class="step_instructions step_highlight">Currently selected: <span id="filter_current_list"></span></div>
             <div id="filter_accordion">
             
+                <h4><a href="#">User Group</a></h4>
+                <div><select id="group" name="group">
+                    {% for group in groups %}
+                    <option value="{{ group.name }}" id="{{ group.name }}">{{ group.name }}</option>
```

Comment:
If you put `group.id` as the `value`, then you can just do `groups` instead of `groups__name` above, and it'll be more robust to naming collisions and possibly faster.  You can still use `group.name` for the dropdown so it'll be the same for users.  Also, I wouldn't bother adding an `id` to the `<option>` unless you for some reason need one; if you do, you should certainly make it more specific (e.g. `user-search-group-{{group.name}}`) since `id`s should be unique in the entire page.